packages/docs/src/pages/qwikcity/content/mdx.mdx

```diff
@@ -1,3 +1,66 @@
 ---
-title: TODO
+title: Qwikcity - MDX
 ---
+
+An alternative way to author content is using `.mdx` files. (Markdown JSX.) These files are authored as Markdown, but they are compiled down to Qwik components. In addition to per Markdown, they can also refer to other components.
+
+Let's assume you have the router set up such as this:
+```
+- src/
+  - routes/
+    - some/           
+      - path/
+        - index.mdx              # http://server/some/path
+```
+
+File: `src/routes/some/path/index.mdx`:
+```Markdown
+---
+title: Hello World Title
+---
+
+This is a simple hello world component.
+
+```
+
+The above component will be rendered at `http://server/some/path`. 
+
+## Importing other components.
+
```

Comment:
[MDX](https://mdxjs.com/) is a creation opportunity for you to come up with new content quickly ("qwikly" 🙂) and if you need more interaction on your page you can seamlessly integrate your Qwik-components.

Reply:
integrated